app/models/user.rb

```diff
@@ -78,9 +77,7 @@ def nodes
   end
 
   def notes
-    Node.where(uid: uid)
-      .where(type: 'note')
-      .order('created DESC')
+    Node.where(uid: uid).where(type: 'note').order('created DESC')
```

Comment:
Actually i prefer the multi-line version because sometimes we change only one element and then it shows up more clearly in the blame -- would you mind changing back? Thank you so much @siaw23 !!!